features/minion/core/api/pom.xml

```diff
@@ -22,6 +22,7 @@
                         <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Version>${project.version}</Bundle-Version>
+                        <Export-Package>org.opennms.minion.core.api.*;version="${project.version}"</Export-Package>
```

Comment:
The default behavioiur is as following: 

> <Export-Package> is now assumed to be the set of packages in your local Java sources, excluding the default package '.' and any packages containing 'impl' or 'internal'. (before version 2 of the bundleplugin it was based on the symbolic name)

Source: http://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html